content/en/tracing/api_catalog/_index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Datadog API Catalog
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+{{< beta-callout url="https://datadoghq.com" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+  Datadog API Catalog is in beta! Use this form request access. 
+{{< /beta-callout >}} 
+
+{{< img src="tracing/api_catalog/api-catalog-overview.png" alt="API Catalog showing a list of API endpoints along with their Error Rate, P99 Latency, Requests Count, and Owning Team, with searching, filtering, and scoping features." style="width:100%;" >}}
+
+## Overview
+
+The API Catalog provides a single view and entry point into exploring the performance, reliability, and ownership of all your API Endpoints in one place. It's a central place where your whole company can find constantly up-to-date information about the characteristics of the APIs used by internal services (private APIs) and external users (publicly exposed APIs). 
+
+Monitor your mission-critical API-driven business features by continuously discovering and tracking the performance of the APIs that you expose to your customers, and the ones your services rely on to function. Standardize performance expectations for APIs and constantly validate these expectations, alerting when performance deviates from them.
+
+This enables your efforts to:
+- Provide high availability and uptime of critical business features.
+- Have fewer regressions and instability issues.
+- Triage incidents quicker.
+- Optimize resources allocation.
+
+API Catalog brings together data from across Datadog provide opinionated workflows so you can explore and monitor your APIs from different sources in one unified view. API Catalog provides:
+
+- **Automated discoverability** - One inventory for all public, private, and partner APIs, where _endpoints_ are the organizing principle.
+- **Correlation of all API metadata** from different Datadog sources.
+- **API endpoint metrics** such as Last Seen, Requests, Latency, and Errors, to identify performance issues and track API health.
+- **Monitoring and governance workflows** such as:
+  - Alerting on Endpoints that deviate from defined performance expectations and thresholds.
+  - Resolving incidents faster with API ownership information directly associated with each endpoint, to know who to reach when something goes wrong
+
+### Some background terminology
```

Comment:
can we move the background terminology section to the end and just have a sentence in the overview that says something like "learn more / APIs terminology" and links to that section in the end?

it is way to heavy to put in the beginning of the docs